internal/page/certificateprovider/onelogin_identity_details.go

```diff
@@ -0,0 +1,60 @@
+package certificateprovider
+
+import (
+	"net/http"
+
+	"github.com/ministryofjustice/opg-go-common/template"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/actor"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/page"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/validation"
+)
+
+type oneloginIdentityDetailsData struct {
+	App                 page.AppData
+	Errors              validation.List
+	CertificateProvider *actor.CertificateProviderProvidedDetails
+}
+
+func OneloginIdentityDetails(tmpl template.Template, certificateProviderStore CertificateProviderStore, lpaStoreResolvingService LpaStoreResolvingService, lpaStoreClient LpaStoreClient) page.Handler {
```

Comment:
I saw you did this with the donor one, could we use `OneLogin`/`one_login` instead as it looks a bit nicer and matches our previous pages. Then we can reserve the single word style for the package